Create ui-c8y-1021-49-12-search-not-highlighting-the-first-element-8482-graft-release-cd-8483.md (#3062)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-49-12-search-not-highlighting-the-first-element-8482-graft-release-cd-8483.md

---
date: ""

> title: Search highlights the first element not anymore

Reviewer: ```suggestion
title: Search no longer highlights the first element
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62914
version: 1021.49.12
---

> In certain scenarios, the search functionality in the UI did highlight the first element in the search results. This has now been fixed. The first element in the search results is now not focused. This improves the user experience when searching for a general search term.

Reviewer: ```suggestion
In certain scenarios, the search functionality in the UI highlighted the first element in the search results. This has  been fixed and the first element in the search results is no longer focused. 
```
